hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -358,6 +358,31 @@ public void setWaitOnAllFollowers(boolean val) {
   )
   private int rocksdbMaxOpenFiles = ROCKSDB_MAX_OPEN_FILES_DEFAULT;
 
+  @Config(key = "rocksdb.auto-compaction-small-sst-file",
```

Comment:
looks like these props should be described in ozone-default.xml

Reply:
@myskov these config annotations are generated at compile time into XML format (search for `ozone-default-generated.xml` after build).  They should not be added manually to `ozone-default.xml`.

Reply:
hmm just for myself learning~ when do we usually add configuration parameters into file _ozone-default.xml_, and when to add to the _Datanodeconfiguration_ class~?

Reply:
Per my understanding,  annotation style configuration is preferred over ozone-default.xml currently.  @DaveTeng0 .

Reply:
got it!! Thanks sammi!!